src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptAsynchronousTaggerProvider.cs

```diff
@@ -19,16 +19,26 @@ internal abstract class VSTypeScriptAsynchronousTaggerProvider<TTag> : Asynchron
         IThreadingContext threadingContext,
         IAsynchronousOperationListenerProvider asyncListenerProvider,
         VSTypeScriptGlobalOptions globalOptions)
-        : base(threadingContext, globalOptions.Service, visibilityTracker: null, asyncListenerProvider.GetListener(FeatureAttribute.Classification))
+        : this(
+            threadingContext,
+            globalOptions,
+            visibilityTracker: null,
+            asyncListenerProvider)
     {
     }
 
+    [Obsolete("Use constructor that takes a single TaggerHost")]
```

Comment:
binary compat overload for TS.  communicated with Maria on moving to the new method once it is inserted (i'll likely make the PR for them).